src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -101,6 +101,22 @@ int MPIDI_OFI_handle_cq_error(int vni, int nic, ssize_t ret);
     } while (_ret == -FI_EAGAIN);                           \
     } while (0)
 
+#define MPIDI_OFI_CALL_RETRY_RETURN(FUNC,vci_,ret) \
+    do { \
+        int _retry = MPIR_CVAR_CH4_OFI_MAX_EAGAIN_RETRY; \
+        while (1) { \
+            ret = FUNC; \
+            if (likely(ret != -FI_EAGAIN)) { \
+                break; \
+            } \
+            _retry--; \
+            MPIR_ERR_CHKANDJUMP(_retry == 0, mpi_errno, MPIX_ERR_EAGAIN, "**eagain"); \
```

Comment:
Same comment as here: https://github.com/pmodels/mpich/pull/6463#discussion_r1150962668

Reply:
I just saw this comment is unresolved. I'm going to push a fixup commit that modifies the retry logic to match what is in the linked PR.